instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/trace/TracingContextUtils.java

```diff
@@ -77,22 +74,4 @@ public static Span getSpanWithoutDefault(
         io.opentelemetry.trace.Span.fromContextOrNull(agentContext);
     return agentSpan == null ? null : toApplication(agentSpan);
   }
-
-  public static Scope currentContextWith(Span applicationSpan) {
```

Comment:
I just deleted this since I think even previously, this wasn't required because of instrumentation of withSpan and makeCurrent